ext/pdo_firebird/firebird_driver.c

```diff
@@ -628,6 +628,8 @@ static bool firebird_handle_preparer(pdo_dbh_t *dbh, zend_string *sql, /* {{{ */
 	do {
 		isc_stmt_handle s = PDO_FIREBIRD_HANDLE_INITIALIZER;
 		XSQLDA num_sqlda;
+		unsigned int i;
+		XSQLVAR* var;
```

Comment:
It would probably be easier to read if put these declarations just before the for loop.

Reply:
done